samples/single-page-app/package.json

```diff
@@ -50,8 +50,8 @@
     "react-dom": "^16.13.1",
     "resolve": "1.15.0",
     "resolve-url-loader": "^4.0.0",
-    "sass-loader": "8.0.2",
-    "semver": "6.3.0",
+    "sass-loader": "9.0.0",
```

Comment:
updated to 12.3.0

https://github.com/fullstorydev/fullstory-babel-plugin-annotate-react/blob/cc043d92478a3a109bf7a75a82f8a240c4e9a675/samples/single-page-app/package.json#L48